ena-submission/src/ena_deposition/upload_external_metadata_to_loculus.py

```diff
@@ -26,7 +26,34 @@
 logger = logging.getLogger(__name__)
 
 
-def get_external_metadata(db_config: SimpleConnectionPool, entry: dict[str, Any]) -> dict[str, Any]:
+def get_results(
```

Comment:
Unclear fn name: which results from where?

Reply:
I initially did that I thought this was more compact (I used your PR for the timestamps as inspiration actually). The name comes from the fact that we are getting the contents of the results_column - but we could make it clear